### ISSUE TITLE
Feature Request: add option to use AUR metadata-server instead of FAUR

### ISSUE THREAD

**Issue by tristanRW** [fosskers/aura#926]
something like: --use-AUR-metadata or --metadataserver where you can set it to a custom link

It would be a nice to have even if it's feature-restricted due to the extra endpoints in FAUR not being available

**Comment by fosskers** [comment]
Thanks for the suggestion.

> due to the extra endpoints in FAUR not being available

Could you elaborate on this?

**Comment by tristanRW** [comment]
Thank you for the swift reply.
So what I would hope for is a option to query the original metadataserver. An example for a use-case could be to make sure a package can be retrieved during development, or simply get a certain update immediately if that is needed.
What I was saying regarding the API was that I am not sure if the "normal" AUR endpoints allow all of auras features to be used. So I suggested implementing the option with reduced features if needed.

**Comment by fosskers** [comment]
>  I am not sure if the "normal" AUR endpoints allow all of auras features to be used

That's correct - the Faur only has a single endpoint, but it accepts query parameters to perform different search types. A one-for-one swap wouldn't be possible if the user of the proposed `--metadataserver` option supplied something that didn't support the same API.

Is that what you were imagining, a one-for-one swap for all API calls?

**Comment by tristanRW** [comment]
But would it be possible to match requests to the API if the original Aur metadataserver? Then a `--useaur` would be possible.

**Comment by fosskers** [comment]
If I changed how the requests were structured internally, yes. I'll ask again though: what's the end goal?

**Comment by tristanRW** [comment]
One usage-scenario would be the ability to have certain packages as soon as they are on the Aur.
It could also benefit resilience, since users could fall back to this option if Faur is not reachable.

**Comment by fosskers** [comment]
> to have certain packages as soon as they are on the Aur.

Right, at the moment you'd have to wait an average of half an hour for the new data to become available, since the Faur refreshes itself every hour. So far this has been enough for people, but developers testing out their packages might have different needs, as you point out.

> users could fall back to this option if Faur is not reachable.

So far the Faur has had no downtime, and the traffic from Aura is far less than what would be required to overload it. I'm willing to wait to see what might organically break it (if anything).

**Comment by tristanRW** [comment]
It's not specifically about Faur possibly breaking at some point. Just the feature to not be forced to use it would be nice. Especially since aura does so many things right that other aur-helpers that just use Aur do not provide.

**Comment by fosskers** [comment]
I will think about how this could possibly be added.
